group_vars/all/all.yml

```diff
@@ -193,3 +193,26 @@ TMP_directories:
 - path: "/srv/galaxy/var/tmp"
   host: "galaxy.atgm.avans.nl"
   time: "2d"
+
+
+#sudoers file
+sudoers_sudoers:
+  defaults:
+    - env_reset
+    - exempt_group=sudo
```

Comment:
not sure we need this line, it doesn't match my local sudoers, does it match the one on the servers?

Reply:
if i remember correctly this was on our servers.
but if it is managed by DIFF, should it not be usefull for the other machines incase some one make a big oeps?

I propose we might test this out on 7kingdoms once i,m ready with backing everything up. if it fails we can easily wipe the machine and reinstall what i was planning to do anyhow.